pubber/src/main/java/daq/pubber/Configuration.java

```diff
@@ -15,8 +17,7 @@ public class Configuration {
   public String keyFile = "local/rsa_private.pkcs8";
   public byte[] keyBytes;
   public String algorithm = "RS256";
-  public Object extraField;
   public String serialNo;
   public String macAddr;
-  public String extraPoint;
+  public HashMap<String, String> options = new HashMap<>();
```

Comment:
I would make this an explicit Options object. The reason is once you go to a general map here you lose the ability to have any checking on the contents and types.  Ideally, this structure would be specified as a protobuf or json schema itself, so we get the auto-documenting process with that.  It also makes the code simpler since it's not a lot of key-lookups-in-maps and makes the code itself more stable through any changes.

Reply:
Turned options into an options object